### ISSUE TITLE
Built-in help subcommand does not work with aliased git-style commands

### ISSUE THREAD

**Issue by MattTheNub** [tj/commander.js#948]
# Example

```js
#!/usr/bin/env node

const program = require('commander');

program
  .version('0.0.1')
  .description('Fake package manager')
  .command('install [name]', 'install one or more packages').alias('i');

program.parse(process.argv);
```
When running the built-in help command on the alias (`pm help i`), this happens:
## Expected Behavior
The alias gets resolved to `install`, and the information is read from `pm-install`
## Actual Behavior
The command attempts to read `pm-i` and fails, as it does not exist.

**Comment by shadowspawn** [comment]
The `npm help foo` pattern has never been documented in the README (#892) and would appear to be buggy making me wonder how implemented it was.

With your sample program it appears to be working as expected for `--help` though, given I only created index.js:

```
$ node index.js install --help
error: index-install(1) does not exist, try --help
$ node index.js i --help
error: index-install(1) does not exist, try --help

```

**Comment by shadowspawn** [comment]
I came across the (simple) implementation code today and had another look at behaviour. Commander is blindly turning "help foo" into "foo --help", but reasonable it should work with alias. 

Categorising this as a bug.

**Comment by shadowspawn** [comment]
This is fixed in the published pre-release of Commander v5.0.0 (#1163)

**Comment by shadowspawn** [comment]
Commander v5.0.0 has been released.

https://github.com/tj/commander.js/releases/tag/v5.0.0
